site/src/editor/mod.rs

```diff
@@ -1011,13 +1011,12 @@ pub fn Editor<'a>(
                     </div>
                     <button
                         class="info-button"
-                        data-title="\
-shift Enter   - Run + Format
- ctrl /       - Toggle line comment
-  alt Up/Down - Swap lines
-shift Delete  - Delete lines
- ctrl Z       - Undo
- ctrl Y       - Redo"
+                        data-title="\ shift Enter   - Run + Format
```

Comment:
The `\` is no longer needed here. You would be trying to escape the space.

Reply:
LOL. Gotcha.